eventually/static/src/containers/taskItem/TaskService.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+const rootUrl = '/api/v1/';
+const eventPath = rootUrl + 'events/';
+
+const getTask = (eventId,taskId, full_name) => {
```

Comment:
add whitespace between eventId and taskId

Reply:
Done